### PR TITLE
Hotfix interface closing

### PR DESCRIPTION
**Pull request by jworcestBAH**
This fixes the issue with connections staying open

### CODE REVIEW

File: tests/baseTestAPI.py [hunk 1]
```diff
@@ -176,6 +176,8 @@ def setUpClass(cls):
 
     def setUp(self):
         """Set up broker unit tests."""
+        # Repopulate interfaces if needed
+        self.interfaces = InterfaceHolder()
```
Comment (nmonga91) [inline tests/baseTestAPI.py]: Is there a reason this is using `InterfaceHolder()` vs `BaseInterface.interfaces` as the broker was using?

Comment (jworcestBAH) [inline tests/baseTestAPI.py]: InterfaceHolder() is the more general option, if BaseInterface.interfaces is populated it copies that, if not it creates new interface objects.

**Comment by nmonga91** [comment]
Looks good, merging!

**Comment by nmonga91** [comment]
Looks good, merging once jenkins checks are complete

**Comment by klrBAH** [comment]
[run tests]

**Comment by nmonga91** [comment]
[run tests]

**Pull request merged**